### DISCUSSION TITLE
Audio waveforms

### DISCUSSION THREAD

**Discussion by Ninko85** [mifi/lossless-cut#1590] [Q&A]
Hi,
When opening a file with two audio tracks, one has 5.1 channels, the other has 2 channels.
The waveform only shows 6 waveforms, which I assume is showing the 5.1 channel track?

Couple of questions,
Which waveform represent which channel?
How do you view the 2 channel track as well, as I would like to compare the two audio tracks?

Thanks

**Comment by mifi** [comment]
Hi

> Which waveform represent which channel?

Not sure. Maybe ffmpeg docs describes the order: https://trac.ffmpeg.org/wiki/Waveform#MultipleChannels

> How do you view the 2 channel track as well, as I would like to compare the two audio tracks?

not yet possible, I added a feature request here: #260
